x-pack/legacy/plugins/uptime/server/lib/helper/__test__/assert_close_to.test.ts

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { assertCloseTo } from '../assert_close_to';
+
```

Comment:
i like tests for a test helped function, it's like nice little inception going on here ;)